javascript/src/containers/SpeciesStep.js

```diff
@@ -1,40 +1,43 @@
-import React, { Component } from 'react';
+import React, { Component } from 'react'
 import { connect } from 'react-redux'
-import { bindActionCreators } from 'redux';
+import { bindActionCreators } from 'redux'
 import { setSpecies } from "../actions/configuration"
 
+
 class SpeciesStep extends Component {
     render() {
+        let species = this.props.species
         return(
-        <div className="configuration-step">
-            <h4 className="title is-5"><span className="badge">1</span> Select Species</h4>
-            <div className="step-content">
-                <div className="select">
-                    <select
-                        value={this.props.configuration.species}
-                        onChange={(e) => this.props.setSpecies(e.target.value)}
-                    >
-                        <option value="none">Select</option>
-                        <option value="lodgepole_pine">Lodgepole Pine</option>
-                        <option value="douglas-fir">Douglas-fir</option>
-                        <option value="sitka_spruce">Sitka Spruce</option>
-                        <option value="ponderosa_pine">Ponderosa Pine</option>
-                        <option value="engelmann_spruce">Engelmann Spruce</option>
-                    </select>
+            <div className="configuration-step">
+                <h4 className="title is-5"><span className="badge">1</span> Select Species</h4>
+                <div className="step-content">
+                    <div className="select">
+                        <select
+                            value={species}
+                            onChange={(e) => this.props.onChange(e.target.value)}
+                        >
+                            <option value="none">Select</option>
+                            <option value="lodgepole_pine">Lodgepole Pine</option>
```

Comment:
For consistency, it would probably be better to use abbreviations for the value (display should still be common name). That way there's minimal translation between the option value, and how it's used to load tiles, etc.